src/pages/volunteer/index.tsx

```diff
@@ -114,24 +114,27 @@ export default () => {
       />
       <Spacer heightDesktop="120px" heightMobile="0" />
       <TwoParagraphBlock
-        title="What it’s like to volunteer at ShelterTech"
-        paragraph1="No matter what role you play, the essential attitude you need is a deep belief in digital equity for all people."
+        title="Volunteering at ShelterTech"
+        paragraph1=""
         paragraph2={
           <>
             <p>We ask that you consider these three things before joining:</p>
             <p>
-              &ndash; You are willing to volunteer at least three hours each
-              week.
+              &bull; You are willing to volunteer at least three hours each week
             </p>
             <p>
-              &ndash; You will be kind, considerate, and ethical towards other
-              volunteers and the communities we serve.
+              &bull; You will be kind, considerate, and ethical towards other
+              volunteers and the communities we serve
             </p>
             <p>
-              &ndash; You understand that ShelterTech is an all-volunteer team,
+              &bull; You understand that ShelterTech is an all-volunteer team,
               and that you will try your best to always represent ShelterTech in
               a professional and thoughftul manner.
```

Comment:
Nit: It looks like you removed the trailing period from every list item except for this one. If we're going to remove the period to match the designs, then we better do it for each item.